flink-ai-flow/lib/airflow/airflow/contrib/jobs/event_based_scheduler_job.py

```diff
@@ -311,7 +313,7 @@ def _schedule_task(self, scheduling_event: TaskSchedulingEvent):
             scheduling_event.execution_date,
             scheduling_event.try_number
         )
-        self.executor.schedule_task(task_key, scheduling_event.action)
+        self.executor.schedule_task(task_key, scheduling_event.action, self.server_uri)
```

Comment:
Change server_uri to the parameter of executor.

Reply:
Fixed in 4147015